helm/action/create.go

```diff
@@ -0,0 +1,34 @@
+package action
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+)
+
+import (
+	"github.com/deis/helm/helm/log"
+)
+
+// Create a chart
+//
+// - chartName being created
+// - homeDir is the helm home directory for the user
+func Create(chartName, homeDir string) {
+
+	skeletonDir, _ := filepath.Abs("skel")
+	if fi, err := os.Stat(skeletonDir); err != nil {
+		log.Die("Could not find %s: %s", skeletonDir, err)
+	} else if !fi.IsDir() {
+		log.Die("Malformed skeleton: %s: Must be a directory.", skeletonDir)
+	}
+
+	chartDir := path.Join(homeDir, "workspace", "charts", chartName)
+
+	// copy skeleton to chart directory
+	if err := copyDir(skeletonDir, chartDir); err != nil {
+		log.Die("failed to copy skeleton directory: %v", err)
+	}
+
+	return
```

Comment:
An empty `return` here isn't needed, and it made me re-scan the function signature to see if there were actually named return variables.